crates/swc_ecma_minifier/tests/fixture/issues/2257/full/output.js

```diff
@@ -8525,7 +8525,7 @@
             }), redefine(URLSearchParamsPrototype, ITERATOR, URLSearchParamsPrototype.entries, {
                 name: "entries"
             }), redefine(URLSearchParamsPrototype, "toString", function() {
```

Comment:
Original code is

```js
redefine(
                URLSearchParamsPrototype,
                "toString",
                function toString() {
                    var entries = getInternalParamsState(this).entries;
                    var result = [];
                    var index = 0;
                    var entry;
                    while (index < entries.length) {
                        entry = entries[index++];
                        result.push(
                            serialize(entry.key) + "=" + serialize(entry.value)
                        );
                    }
                    return result.join("&");
                },
                { enumerable: true }
            );

```

Reply:
```js
                        entry = entries[index++];
                        result.push(
                            serialize(entry.key) + "=" + serialize(entry.value)
                        );
```

`entry = entries[index++];` is inlined

Reply:
It make sense as `result.push` (member property access) does not modify `entry` or `entries[index++]`.